### PR TITLE
Fix for show/hide unlock

### PR DESCRIPTION
**Pull request by Aca-jov**

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

**Comment by pavlovicnemanja** [comment]
:+1:

File: scripts/superdesk-authoring/views/authoring-header.html [hunk 1]
```diff
@@ -212,7 +212,8 @@
             ng-if="cv.terms.length && schema[cv._id]"
             sd-width="{{editor[cv._id].sdWidth}}"
             order="{{editor[cv._id].order}}">
-          <label class="authoring-header__item-label">{{ cv.display_name | translate }}</label>
+            {{schema[cv._id].required}}
+          <label class="authoring-header__item-label" translate>{{ cv.display_name }}</label>
```
Comment (vladnicoara) [inline scripts/superdesk-authoring/views/authoring-header.html]: This will break the translation.

**Comment by vladnicoara** [comment]
LGTM except the translation issue

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**